File: tests/corpus/positive-fixtures/QA-CS-107/FeedTests.cs
using Microsoft.Playwright;
using Microsoft.Playwright.MSTest;
using Microsoft.VisualStudio.TestTools.UnitTesting;

[TestClass]
public class FeedTests
{
    [TestMethod]
    public async Task LoadsFeed()
    {
        await Page.GotoAsync("https://example.com/feed");
        await Page.WaitForLoadStateAsync(LoadState.NetworkIdle);
        Assert.IsNotNull(Page);;
    }

    [TestMethod]
    public async Task LoadsTimeline()
    {
        await Page.WaitForLoadStateAsync(LoadState.NetworkIdle);
        Assert.IsNotNull(Page);;
    }

    [TestMethod]
    public async Task LoadsInbox()
    {
        await Page.WaitForLoadStateAsync(LoadState.NetworkIdle);
        Assert.IsNotNull(Page);;
    }

    [TestMethod]
    public async Task LoadsArchive()
    {
        await Page.WaitForLoadStateAsync(LoadState.NetworkIdle);
        Assert.IsNotNull(Page);;
    }
}

File: tests/corpus/positive-fixtures/QA-JV-107/FeedTests.java
import com.microsoft.playwright.*;
import com.microsoft.playwright.options.LoadState;
import org.junit.jupiter.api.Test;

public class FeedTests {

    @Test
    public void loadsFeed(Page page) {
        page.navigate("https://example.com/feed");
        page.waitForLoadState(LoadState.NETWORKIDLE);
    }

    @Test
    public void loadsTimeline(Page page) {
        page.waitForLoadState(LoadState.NETWORKIDLE);
    }

    @Test
    public void loadsInbox(Page page) {
        page.waitForLoadState(LoadState.NETWORKIDLE);
    }

    @Test
    public void loadsArchive(Page page) {
        page.waitForLoadState(LoadState.NETWORKIDLE);
    }
}
